model/project.go

```diff
@@ -1778,11 +1778,11 @@ func (p *Project) IsGenerateTask(taskName string) bool {
 }
 
 func findAliasesForPatch(projectId, alias string, patchDoc *patch.Patch) ([]ProjectAlias, error) {
-	vars, shouldExit, err := findAliasInProjectOrRepoFromDb(projectId, alias)
+	vars, err := findAliasInProjectOrRepoFromDb(projectId, alias)
 	if err != nil {
 		return nil, errors.Wrap(err, "getting alias from project")
 	}
-	if !shouldExit && len(vars) == 0 {
+	if len(vars) == 0 {
```

Comment:
I know you didn't write this but can we instead say on this line `if len(vars) > 0 { return vars }` and then remove the need for a big indentation below for readability purposes

Reply:
done